### ISSUE TITLE
[Feature Req] Open rather than play Playlist

### ISSUE THREAD

**Issue by rodalpho** [yuliskov/SmartTubeNext#2033]
TL;DR: Request is to allow users to configure whether selecting a playlist opens or immediately starts playing it.

Bit more detail:
A common use-case for me is to catch a video from the algorithm, go into its channel, and scroll down to their saved playlists to see what else they've got that's similar to what I liked. 

In previous versions of SmartTube when I selected a playlist it would open, listing all the videos inside. As of a couple months back the default action is now to play the whole thing. I can still open it instead with a long-press and selecting from the context menu, but I would prefer to revert to the old behavior. Can this be made configurable?

**Comment by rodalpho** [comment]
This actually started working again at some point. Unclear if it was a config change on my end or a code change. Either way, all good.
